<ui version="4.0" >
 <class>dEditCat</class>
 <widget class="Q3MainWindow" name="dEditCat" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Catalogue</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>297</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget2" >
      <property name="font" >
       <font/>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="Label" >
            <property name="text" >
             <string>Name:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eName" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3" >
          <property name="text" >
           <string>Description:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Q3TextEdit" name="eDescription" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Aliases</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tAliases" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>2</number>
          </property>
          <column>
           <property name="text" >
            <string>Lang</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Element string view</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="1" column="0" >
         <widget class="QSEditor" native="1" name="eStrViewF" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QComboBox" name="eSv" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Group string view</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="1" column="0" >
         <widget class="QSEditor" native="1" name="eStrViewFG" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QComboBox" name="eSvG" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Access</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tRoles" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>2</number>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
          <property name="selectionMode" >
           <enum>Q3Table::Multi</enum>
          </property>
          <column>
           <property name="text" >
            <string>Read</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Write</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>QSEditor</class>
   <extends>QWidget</extends>
   <header>qseditor.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mdtree.h</include>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >mainform.h</include>
  <include location="local" >roleeditor.h</include>
  <include location="local" >qseditor.h</include>
  <include location="local" >qseditor.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>eSv</sender>
   <signal>activated(int)</signal>
   <receiver>dEditCat</receiver>
   <slot>eSv_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eSv</sender>
   <signal>highlighted(int)</signal>
   <receiver>dEditCat</receiver>
   <slot>eSv_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eSvG</sender>
   <signal>activated(int)</signal>
   <receiver>dEditCat</receiver>
   <slot>eSvG_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eSvG</sender>
   <signal>highlighted(int)</signal>
   <receiver>dEditCat</receiver>
   <slot>eSvG_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
